Code/Framework/AzQtComponents/AzQtComponents/Components/Widgets/ElidingLabel.cpp

```diff
@@ -8,12 +8,15 @@
 #include <AzCore/std/algorithm.h>
 #include <AzQtComponents/Components/Widgets/ElidingLabel.h>
 
+#include <QApplication>
 #include <QHBoxLayout>
 #include <QLabel>
 #include <QPainter>
 #include <QStyle>
 #include <QTextCursor>
 #include <QTextDocument>
+#include <QThread>
+#include <QTimer>
```

Comment:
These added includes `QApplication` `QThread` and `QTimer` can be removed now that they aren't used in the new implementation.

Reply:
Bother, thanks: fixed.